tekton/release-cheat-sheet.md

```diff
@@ -98,36 +98,28 @@ the triggers repo, a terminal window and a text editor.
     TEKTON_PACKAGE=tektoncd/triggers
     ```
 
-    1. Create a `PipelineResource` of type `git`
-
-    ```shell
-    cat <<EOF | kubectl --context dogfooding create -f -
```

Comment:
Do we no longer need to create a PipelineResource? (That's great!)

Reply:
Indeed, I removed `PipelineResources` - which is why the release draft `Task` is now a `Pipeline` (clone + bucket download + release draf)